spec/facades/book_facade_spec.rb

```diff
@@ -7,8 +7,8 @@
     expect(book.count).to eq(10)
   end
 
-  xit 'returns book by id', :vcr do
+  it 'returns book by id', :vcr do
     all_books = BookFacade.book_show("ZV9DDwAAQBAJ")
```

Comment:
Might be good to call this variable `book` since we are just getting one back?

Reply:
Unsure who the test but I have updated the  variable to make more sense! 
all_books is now book